exercises/tuples-lists-dictionaries/challenge_070.py

```diff
@@ -0,0 +1,41 @@
+# TUPLES, LISTS AND DICTIONARIES
+
+# Challenge 070
+"""
+70: Add to program 069 to ask the user a number and display the country in that position.
+"""
+
+country_names = ("England", "Italy", "Spain", "France", "Portugal")
+
+print(country_names)
+
+
+# Ask for user input and ensures the user enters an integer
+def get_user_input_digit(message):
+    while True:
+        user_input = input(message)
+        if not user_input.isdigit():
```

Comment:
What happens here if there are more than 10 countries in the list and a user wants to display them

Reply:
I have added a parameter for the max index and added a join to convert the list to string